test/functional/architecture_test.rb

```diff
@@ -24,26 +24,6 @@
       result = run_cmd(@cmd)
       assert_cmd(expected_result, result)
     end
-
-    it 'should run list command with defaults' do
```

Comment:
Why are you removing tests like that? I find them pretty useful. Although I'd change the version `2.1` to the latest one.

Reply:
Yes I agree, I think the API calls are different now but I will figure out the correct calls and fix the tests.